hack/e2e/run.sh

```diff
@@ -146,5 +146,6 @@ ginkgo . -p -nodes="${GINKGO_NODES}" -v --focus="${GINKGO_FOCUS}" --skip="${GINK
 popd
 
 if [[ "${DOWN}" = "yes" ]]; then
-    ${test_run}/kops delete cluster --name "${CLUSTER_NAME}" --yes
+    # This should be changed to ${test_run}/kops once https://github.com/kubernetes/kops/pull/13217 is merged.
+    ${test_run}/${test_run_id}/kops delete cluster --name "${CLUSTER_NAME}" --yes
```

Comment:
` ${KUBETEST2} --kops-binary-path="${test_run}/${test_run_id}/kops" --down || echo "kubetest2 down failed"`
This command may be more desirable (maybe kops-binary-path also should use runpath by default?). This one will dump cluster logs and such, which can be quite useful. E.g https://gcsweb.k8s.io/gcs/kubernetes-jenkins/logs/e2e-kops-pipeline-updown-kops124/1562140270143737856/artifacts/cluster-info/